SalesforceSDK/Core/Logging/ILoggingService.cs

```diff
@@ -26,12 +26,13 @@
  */
 
 using System;
+using System.Runtime.CompilerServices;
 
 namespace Salesforce.SDK.Logging
 {
     public interface ILoggingService
     {
-        void Log(string message, LoggingLevel loggingLevel);
-        void Log(Exception exception, LoggingLevel loggingLevel);
+        void Log(string message, LoggingLevel loggingLevel, [CallerMemberName]string memberName = "", [CallerFilePath]string classPath = "", [CallerLineNumber]int line = 0);
```

Comment:
I would change classPath to filePath since that's what it really is, even though we use it in place of class path.